### DISCUSSION TITLE
Is the "to_anki" function working?

### DISCUSSION THREAD

**Discussion by FranciscoFrutos** [paulbricman/autocards#34] [Q&A]
It throws the 

"Card template ⁨1⁩ in notetype '⁨Autocards⁩' has a problem.brExpected to find a field replacement on the front of the card template.
22 cards were not sent correctly."

 exception 

or am i missing a step?

**Comment by thiswillbeyourgithub** [comment]
Please show your anki version information, screenshots of the error, more context.
